src/stars.css

```diff
@@ -0,0 +1,91 @@
+.tutto {
+  height: 160px;
+  width: 140px;
+}
+
+.stars {
+  display: flex;
+  margin-left: 20px;
+  margin-right: 20px;
+}
```

Comment:
Try to use padding because we can not set outer styles, positions and size properties for BEM Blocks.

Reply:
My fault is getting used to over-using margins. 
I deal with almost all distances with margins. 
I need to adopt with "position", "height&width", "paddings" and ... 
in the new stars.css I only used margins 2 times and everything working magically